### PR TITLE
Change python segmentation demo

### PR DESCRIPTION
**Pull request by akorobeinikov**
This PR is a try to solve issue #411. Features:
1. Now demo can work with files and save results to ouput file.
2. Demo using numpy random to create colors for classes.
3. Users can specify classes and colors.
4. Add parameter `classes` to add information about accordance class-color to the image.

### CODE REVIEW

**Comment by akorobeinikov** [comment]
Jenkins ok to test

File: demos/python_demos/segmentation_demo/segmentation_demo.py [hunk 1]
```diff
@@ -57,6 +58,13 @@ def build_argparser():
                       required=True, type=str)
     args.add_argument("-i", "--input", help="Required. Path to a folder with images or path to an image files",
                       required=True, type=str, nargs="+")
+    args.add_argument("-c", "--classes", help="Optional. Path to a text file containing class labels.",
```
Comment (eizamaliev) [inline demos/python_demos/segmentation_demo/segmentation_demo.py:61]: Please, rename it to `--labels`.

File: demos/python_demos/segmentation_demo/segmentation_demo.py [hunk 2]
```diff
@@ -57,6 +58,13 @@ def build_argparser():
...
+                      type=str)
+    args.add_argument("-col", "--colors", help="Optional. Path to a text file containing colors for classes.",
```
Comment (eizamaliev) [inline demos/python_demos/segmentation_demo/segmentation_demo.py:63]:
```suggestion
    args.add_argument("-c", "--colors", help="Optional. Path to a text file containing colors for classes.",
```

Comment (eizamaliev) [reply demos/python_demos/segmentation_demo/segmentation_demo.py:63]: @Wovchena any objections?

**Review by eizamaliev — CHANGES_REQUESTED** [review]

File: demos/python_demos/segmentation_demo/segmentation_demo.py [hunk 3]
```diff
@@ -124,16 +172,38 @@ def main():
         _, _, out_h, out_w = res.shape
     else:
         raise Exception("Unexpected output blob shape {}. Only 4D and 3D output blobs are supported".format(res.shape))
+
+    classes = get_info_from_file(args.classes)
```
Comment (eaidova) [inline demos/python_demos/segmentation_demo/segmentation_demo.py:176]:
```
AttributeError: 'Namespace' object has no attribute 'classes'
```

**Review by eizamaliev — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Pull request merged**